src/torch_admp/qeq.py

```
@@ -199,7 +202,7 @@ def __init__(
         eps: float = 1e-4,
         units_dict: Optional[Dict] = None,
         damping: bool = True,
-        sel: list[int] = None,
+        sel: list[int] = [],
```

Ruff (0.14.11)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)